src/layout/layout.js

```diff
@@ -303,7 +303,7 @@
 
           var drawerButtonIcon = document.createElement('i');
           drawerButtonIcon.classList.add(this.CssClasses_.ICON);
-          drawerButtonIcon.textContent = this.Constant_.MENU_ICON;
+          drawerButtonIcon.innerHTML = this.Constant_.MENU_ICON;
```

Comment:
`innerText` doesn’t work because it auto-escapes. I could run a replace on `<` and `>`. Good enough?